modules.d/99systemd-sysusers/module-setup.sh

```diff
@@ -23,4 +23,7 @@ install() {
         set -o pipefail
         systemd-sysusers --root="$initdir" 2>&1 >&3 | grep -v "^Creating " >&2
     } 3>&1
+
+    # delete shadow files as initramfs is not designed to ask for a non-root user password interactively
```

Comment:
```suggestion
    # delete shadow files as initramfs is not designed to ask for a non-root user password
    # interactively and their permissions crashes the build.
```

Reply:
Did you check #1242? It may be "just" a missing permission but there's no image produced. "crash" = showstopper = nothing works.